### ISSUE TITLE
Asterisk selection on the information_schema not longer working

### ISSUE THREAD

**Issue by quodt** [crate/crate#5857]
**CrateDB version**:

2.0.3

**JVM version**:

java version "1.8.0_60"
Java(TM) SE Runtime Environment (build 1.8.0_60-b27)
Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)

**OS version / environment description**:

MacOS Sierra
version: 10.12.3 (16D32)

**Problem description**:

Asterisk selection on the information_schema results in SQLActionException. Selecting one specific column works fine. Doing an asterisk select in older versions (for example 1.0.5) works fine as well.

**Steps to reproduce**:

SELECT * FROM information_schema.tables;

results in:

Error!

SQLActionException[NullPointerException in org.apache.lucene.util.BytesRef.<init>(BytesRef.java:87)]

**Comment by mfussenegger** [comment]
Cannot reproduce this with a single 2.0.3 node.

Can you run the same query again with error-trace enabled and post the output? (``--verbose`` in crash, or marking the ``error trace`` flag in the admin UI.

**Comment by quodt** [comment]
Same query with `--verbose`::

```
cr> SELECT * FROM information_schema.tables;
SQLActionException[NullPointerException in org.apache.lucene.util.BytesRef.<init>(BytesRef.java:87)]

SQLActionException: INTERNAL_SERVER_ERROR 5000 NullPointerException in org.apache.lucene.util.BytesRef.<init>(BytesRef.java:87)
	at org.apache.lucene.util.BytesRef.<init>(BytesRef.java:87)
	at io.crate.operation.reference.information.InformationSchemaExpressionFactories$39.value(InformationSchemaExpressionFactories.java:461)
	at io.crate.operation.reference.information.InformationSchemaExpressionFactories$39.value(InformationSchemaExpressionFactories.java:457)
	at io.crate.operation.InputRow.get(InputRow.java:46)
	at io.crate.data.RowColumns$ProxyInput.value(RowColumns.java:69)
	at io.crate.data.RowBridging$2.get(RowBridging.java:66)
	at io.crate.operation.collect.InputCollectExpression.setNextRow(InputCollectExpression.java:38)
	at io.crate.operation.collect.InputCollectExpression.setNextRow(InputCollectExpression.java:26)
	at io.crate.operation.aggregation.RowTransformingBatchIterator.moveNext(RowTransformingBatchIterator.java:78)
	at io.crate.data.CloseAssertingBatchIterator.moveNext(CloseAssertingBatchIterator.java:55)
	at io.crate.data.LimitingBatchIterator.moveNext(LimitingBatchIterator.java:54)
	at io.crate.operation.aggregation.RowTransformingBatchIterator.moveNext(RowTransformingBatchIterator.java:76)
	at io.crate.data.ForwardingBatchIterator.moveNext(ForwardingBatchIterator.java:50)
	at io.crate.action.sql.BatchConsumerToResultReceiver.consumeIt(BatchConsumerToResultReceiver.java:61)
	at io.crate.action.sql.BatchConsumerToResultReceiver.accept(BatchConsumerToResultReceiver.java:48)
	at io.crate.executor.transport.executionphases.InterceptingBatchConsumer.tryForwardResult(InterceptingBatchConsumer.java:81)
	at io.crate.executor.transport.executionphases.InterceptingBatchConsumer.accept(InterceptingBatchConsumer.java:68)
	at io.crate.data.ListenableBatchConsumer.accept(ListenableBatchConsumer.java:40)
	at io.crate.operation.projectors.ProjectingBatchConsumer.accept(ProjectingBatchConsumer.java:104)
	at io.crate.operation.collect.BatchIteratorCollectorBridge$AsyncBatchItCollector.invokeConsumer(BatchIteratorCollectorBridge.java:118)
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:760)
	at java.util.concurrent.CompletableFuture.uniWhenCompleteStage(CompletableFuture.java:778)
	at java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:2140)
	at io.crate.operation.collect.BatchIteratorCollectorBridge$AsyncBatchItCollector.doCollect(BatchIteratorCollectorBridge.java:96)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:458)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by Autophagy** [comment]
Hi @quodt - this has been fixed as part of #5873 and will be included in the next 2.0 hotfix. Thank you for reporting!
